ADMINS-FOR-CILIUM.md

```diff
@@ -0,0 +1,18 @@
+# Admins for Cilium
+
+This document lists the individuals responsible for the various bits of infrastructure that support the Cilium project. Ping these folks if you experience issues or need special access.
+
+| Service | Purpose | Admins |
+|---------|---------|--------|
+| [Docker Hub](https://hub.docker.com/u/cilium) | Container registry |
+| [GitHub](https://github.com/cilium) | Source control, bug tracking |
+| [Google Analytics](https://analytics.google.com/analytics/web/) | Analytics for cilium.io | [xmulligan](https://github.com/xmulligan)
+| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for cilium.io | [xmulligan](https://github.com/xmulligan)
+| [Google Domains](https://domains.google/) | Domain registration for cilium.io | [xmulligan](https://github.com/xmulligan)
+| [Google Groups](https://groups.google.com/) | Mailing lists | 
+| [Google Search Console](https://search.google.com/search-console/about) | Search engine analytics for cilium.io | [xmulligan](https://github.com/xmulligan)
```

Comment:
I'm not sure if we actually use this one either

Reply:
I will take it out for now